doc/nrf/ug_partition_manager.rst

```diff
@@ -279,3 +279,72 @@ CMake usage
 
       --slot-size $<TARGET_PROPERTY:partition_manager,MCUBOOT_SLOT_SIZE>
 
+.. _ug_pm_predefined:
+
+Predefined configuration
+========================
+Predefined configuration is a feature intended for deployed products which consists of multiple images where
+only a subset of the included images will be upgradeable through firmware update mechanisms.
+One example of this is a device which includes a non-upgradable first stage bootloader and an upgradeable application.
+In these use cases, the images to be upgraded must be linked to the same address as that which is deployed.
```

Comment:
If you only have 2 images, as this use-case appears to describe, and one is static, isn't the whole thing static?

Maybe we should use a use-case where there are two dynamic partitions?

Reply:
There can only exist one dynamic partition, I will try to explain more clearly.

Reply:
Changed name from dynamic partition to dynamic area to better explain that this is not some actual partition being handled like any other. There exists only one dynamic area, as described further up in this file with the updates.